src/lib/Sendbird.jsx

```diff
@@ -167,6 +169,11 @@ export default function Sendbird(props) {
           pubSub,
           imageCompression,
           useReaction,
+          isMentionEnabled: isMentionEnabled || false,
+          userMention: {
+            maxMentionCount: userMention?.maxMentionCount || 10,
+            maxSuggestionCount: userMention?.maxSuggestionCount || 15,
```

Comment:
nit: moving these into some kind of const is cleaner

Reply:
Thanks for the good catch!